--- origsrc/buzztard-0.4.0/docs/reference/bt-cmd/Makefile.am	2008-09-24 12:57:09.000000000 -0500
+++ src/buzztard-0.4.0/docs/reference/bt-cmd/Makefile.am	2009-03-01 21:28:22.265625000 -0600
@@ -64,9 +64,9 @@ GTKDOC_CFLAGS = \
 	$(BASE_DEPS_CFLAGS) \
 	$(BT_CFLAGS)
 GTKDOC_LIBS = \
+	$(top_builddir)/src/ui/cmd/libbtcmd.la \
 	$(top_builddir)/src/lib/core/libbuzztard-core.la \
 	$(top_builddir)/src/lib/ic/libbuzztard-ic.la \
-	$(top_builddir)/src/ui/cmd/libbtcmd.la \
 	$(BASE_DEPS_LIBS) \
 	$(BT_LIBS)
 
--- origsrc/buzztard-0.4.0/docs/reference/bt-edit/Makefile.am	2008-09-24 12:57:09.000000000 -0500
+++ src/buzztard-0.4.0/docs/reference/bt-edit/Makefile.am	2009-03-01 21:33:00.750000000 -0600
@@ -69,9 +69,9 @@ GTKDOC_CFLAGS = \
 	$(GUI_DEPS_CFLAGS) \
 	$(BT_CFLAGS)
 GTKDOC_LIBS = \
+	$(top_builddir)/src/ui/edit/libbtedit.la \
 	$(top_builddir)/src/lib/core/libbuzztard-core.la \
 	$(top_builddir)/src/lib/ic/libbuzztard-ic.la \
-	$(top_builddir)/src/ui/edit/libbtedit.la \
 	$(BASE_DEPS_LIBS) \
 	$(GUI_DEPS_LIBS) \
         	$(BT_LIBS)
--- origsrc/buzztard-0.4.0/src/ui/cmd/Makefile.am	2008-09-24 12:57:08.000000000 -0500
+++ src/buzztard-0.4.0/src/ui/cmd/Makefile.am	2009-03-01 19:46:56.812500000 -0600
@@ -2,9 +2,9 @@
 
 bin_PROGRAMS=buzztard-cmd
 
-buzztard_cmd_LDADD = $(BASE_DEPS_LIBS) $(BT_LIBS) \
+buzztard_cmd_LDADD = libbtcmd.la \
 	$(top_builddir)/src/lib/core/libbuzztard-core.la \
-	libbtcmd.la
+	$(BASE_DEPS_LIBS) $(BT_LIBS)
 buzztard_cmd_SOURCES= \
 	bt-cmd.c bt-cmd.h
 
--- origsrc/buzztard-0.4.0/src/ui/edit/Makefile.am	2008-09-24 12:57:08.000000000 -0500
+++ src/buzztard-0.4.0/src/ui/edit/Makefile.am	2009-03-01 19:47:19.750000000 -0600
@@ -2,10 +2,10 @@
 
 bin_PROGRAMS=buzztard-edit
 
-buzztard_edit_LDADD = $(BASE_DEPS_LIBS) $(GUI_DEPS_LIBS) $(BT_LIBS) \
+buzztard_edit_LDADD = libbtedit.la \
 	$(top_builddir)/src/lib/core/libbuzztard-core.la \
 	$(top_builddir)/src/lib/ic/libbuzztard-ic.la \
-	libbtedit.la
+	$(BASE_DEPS_LIBS) $(GUI_DEPS_LIBS) $(BT_LIBS)
 buzztard_edit_SOURCES= \
 	bt-edit.c bt-edit.h
 
--- origsrc/buzztard-0.4.0/tests/Makefile.am	2008-11-23 10:55:41.000000000 -0600
+++ src/buzztard-0.4.0/tests/Makefile.am	2009-03-01 21:13:01.265625000 -0600
@@ -42,26 +42,29 @@ libbt_check_la_SOURCES = \
 	bt-test-settings.c bt-test-settings.h
 
 bt_core_LDADD = \
-	$(top_builddir)/src/lib/core/libbuzztard-core.la \
+	libbt-check.la \
 	$(top_builddir)/tests/lib/core/libbtcore-check.la \
-	libbt-check.la $(BASE_DEPS_LIBS) $(BT_LIBS) -lcheck
+	$(top_builddir)/src/lib/core/libbuzztard-core.la \
+	$(BASE_DEPS_LIBS) $(BT_LIBS) -lcheck
 bt_core_LDFLAGS = -Wl,--rpath -Wl,$(top_builddir)/src/lib/core 
 bt_core_SOURCES= \
 	m-bt-core.c
 
 bt_cmd_LDADD = \
-	$(top_builddir)/src/lib/core/libbuzztard-core.la \
+	libbt-check.la \
 	$(top_builddir)/tests/ui/cmd/libbt-cmd-check.la \
-	libbt-check.la $(BASE_DEPS_LIBS) $(BT_LIBS) -lcheck
+	$(top_builddir)/src/lib/core/libbuzztard-core.la \
+	$(BASE_DEPS_LIBS) $(BT_LIBS) -lcheck
 bt_cmd_LDFLAGS = -Wl,--rpath -Wl,$(top_builddir)/src/lib/core 
 bt_cmd_SOURCES= \
 	m-bt-cmd.c
 
 bt_edit_LDADD = \
+	libbt-check.la \
+	$(top_builddir)/tests/ui/edit/libbt-edit-check.la \
 	$(top_builddir)/src/lib/core/libbuzztard-core.la \
 	$(top_builddir)/src/lib/ic/libbuzztard-ic.la \
-	$(top_builddir)/tests/ui/edit/libbt-edit-check.la \
-	libbt-check.la $(BASE_DEPS_LIBS) $(GUI_DEPS_LIBS) $(BT_LIBS) -lcheck
+	$(BASE_DEPS_LIBS) $(GUI_DEPS_LIBS) $(BT_LIBS) -lcheck
 bt_edit_LDFLAGS = \
 	-Wl,--rpath -Wl,$(top_builddir)/src/lib/core \
 	-Wl,--rpath -Wl,$(top_builddir)/src/lib/ic 
